ast2vec/repo2/base.py

```diff
@@ -2,18 +2,19 @@
 import logging
 import multiprocessing
 import os
-from queue import Queue
 import re
 import shutil
 import subprocess
 import tempfile
 import threading
 from collections import namedtuple
+from queue import Queue
```

Comment:
but also I can not find anything about the alphabetical order of imports. 
And find an awesome answer on [StackOverflow](https://stackoverflow.com/a/20763446).
I think we need discuss it and add to our style gide.